[2024-03-07T05:00:38.171+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Carregamento_dos_dados_analitico manual__2024-03-07T04:53:09.087899+00:00 [queued]>
[2024-03-07T05:00:38.185+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Carregamento_dos_dados_analitico manual__2024-03-07T04:53:09.087899+00:00 [queued]>
[2024-03-07T05:00:38.186+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-07T05:00:38.266+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Carregamento_dos_dados_analitico> on 2024-03-07 04:53:09.087899+00:00
[2024-03-07T05:00:38.317+0000] {standard_task_runner.py:60} INFO - Started process 6589 to run task
[2024-03-07T05:00:38.320+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Extraçao_de_dados_do_linkedin', 'Carregamento_dos_dados_analitico', 'manual__2024-03-07T04:53:09.087899+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/dag_exemplo.py', '--cfg-path', '/tmp/tmpod1luvu5']
[2024-03-07T05:00:38.322+0000] {standard_task_runner.py:88} INFO - Job 105: Subtask Carregamento_dos_dados_analitico
[2024-03-07T05:00:38.437+0000] {task_command.py:423} INFO - Running <TaskInstance: Extraçao_de_dados_do_linkedin.Carregamento_dos_dados_analitico manual__2024-03-07T04:53:09.087899+00:00 [running]> on host 4fca1e22de4a
[2024-03-07T05:00:38.611+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Extraçao_de_dados_do_linkedin' AIRFLOW_CTX_TASK_ID='Carregamento_dos_dados_analitico' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T04:53:09.087899+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-07T04:53:09.087899+00:00'
[2024-03-07T05:00:39.623+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-07T05:00:39.630+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Extraçao_de_dados_do_linkedin, task_id=Carregamento_dos_dados_analitico, execution_date=20240307T045309, start_date=20240307T050038, end_date=20240307T050039
[2024-03-07T05:00:39.736+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T05:00:39.751+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
